simpleLooping/test_removeSpaceAtEnd.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+from removeSpaceAtEnd import removeSpaceAtEnd
+
+class TestRemoveSpaceAtEnd(unittest.TestCase):
+    def test_emptyList(self):
+        self.assertEqual(removeSpaceAtEnd(''), '')
```

Comment:
Empty string should be using “”, not ‘’. Only single character using single quote notation.

Reply:
Hello? Did you have a look on this one?

Reply:
Please review this.